docs/recipes/offline-transactions.md

```diff
@@ -1,3 +1,66 @@
 ---
 title: Sending Offline Transactions
----
\ No newline at end of file
+---
+
+# Offline Transaction
+
+## Sign Transaction
+
+Before we start it, I'm going give a outline of transaction.
```

Comment:
Could be more helpful to go over the steps vs listing the transaction outline. We'll have the transaction outline as an ingredient in a separate section, which we can link here.

Reply:
I think you're right. This example seems too long. I will separate it into steps.